package.json

```diff
@@ -17,6 +17,7 @@
   "author": "Abel M. Osorio <abel.m.osorio@gmail.com>",
   "license": "ISC",
   "dependencies": {
+    "@types/dotenv": "^8.2.0",
```

Comment:
What is this package for? If I'm not mistaken, it's for usage with TypeScript, which you're not using right now. Remove it if it's not necessary.